### PR TITLE
Remove fulfillment places

### PR DESCRIPTION
**Pull request by marcoles**
…illmentPlaces

Included changes should be more consistent with the provided code sample. 
Removed the part of tutorial where user would send an outdated request, which apparently does not work right now.

### CODE REVIEW

**Review by DawidJeziorskiGrid — APPROVED** [review]

File: tutorials/retail_api_v2_add_fulfillment_places_java.md [hunk 1]
```diff
@@ -133,62 +128,33 @@ Clone the Git repository with all the code samples to learn the Retail features
...
+1. Go to the `getAddFulfillmentRequest()` method in `AddFulfillmentPlaces` class and check which fields need to be set in order to send the `AddFulfillmentPlacesRequest`:
     - `product`—the product name whose inventory information will be updated
     - `type`—the fulfillment type. You can set one of the [supported values](https://cloud.google.com/retail/docs/reference/rpc/google.cloud.retail.v2#addfulfillmentplacesrequest).
-    - `place_ids[]`—the store IDs for each of the fulfillment types
-    - `add_time`—the time when the fulfillment updates are pushed. It is used to prevent out-of-order updates on the fulfillment information. If this isn't provided, the internal system time will be used.
-    - `allow_missing`—if set to true and the product is not found, the fulfillment information will be retained for up to 24 hours and processed after the product is created.
-
-1. Open <walkthrough-editor-select-regex filePath="cloudshell_open/java-retail/samples/snippets/src/main/java/product/AddFulfillmentPlaces.java" regex="Add fulfillment request">product/AddFulfillmentPlaces.java</walkthrough-editor-select-regex> file and check the `AddFulfillmentPlacesRequest` request.
```
Comment (tetiana-karasova) [inline tutorials/retail_api_v2_add_fulfillment_places_java.md:143]: please dont remove the link - you can place this as a first step here

**Review by tetiana-karasova — APPROVED** [review]

File: tutorials/retail_api_v2_remove_fulfillment_places_java.md [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+   The code samples for each of the Retail services are stored in different directories.
+
+## Remove fulfillment places
+
+1. Before you start, build the Maven project and go to the code samples directory - our starting point to run code samples:
+   ```bash
+   cd ~/java-retail | mvn clean install -DskipTests
+   cd ~/java-retail/samples/snippets  
+   ```
+
```
Comment (t-karasova) [inline tutorials/retail_api_v2_remove_fulfillment_places_java.md:130]: please also add here theOpen <walkthrough-editor-select-regex ...

File: tutorials/retail_api_v2_add_fulfillment_places_java.md (hunk 1 above)
Comment (marcoles) [reply tutorials/retail_api_v2_add_fulfillment_places_java.md:143]: fixed

File: tutorials/retail_api_v2_remove_fulfillment_places_java.md (hunk 2 above)
Comment (marcoles) [reply tutorials/retail_api_v2_remove_fulfillment_places_java.md:129]: fixed

**Review by t-karasova — APPROVED** [review]

**Pull request merged**